packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,98 @@
+### 1: Build a sign-in page on the client
+
+Build a sign-in page that capture's the user's name and password, as shown in the following example.
+
+![Basic sign-in dialog](/img/authenticators/java-authenticators-signinform.png)
+
+### 2: Authenticate the user credentials
+
+After a user initiates the sign-in flow by entering their username and password and then clicking **Sign In**, add the following code:
+
+```java
+// create an IDX Authentication Wrapper (can be an application scoped singleton)
+IDXAuthenticationWrapper idxAuthenticationWrapper = new IDXAuthenticationWrapper();
+
+// begin transaction
+// The proceedContext needs to be persisted between interactions (e.g. stored in a HttpSession for web apps)
```

Comment:
```suggestion
// The proceedContext needs to be persisted between interactions (for example, stored in a HttpSession for web apps)
```

Reply:
The proceed context is only used once on the next request and needs to be updated on each call